tests/ffi/common/prepare.sh

```diff
@@ -4,8 +4,8 @@
 
 DROP_IN_DIR="/etc/containers/systemd/qm.container.d/"
 OSTREE_PATH="/run/ostree"
-export QM_HOST_REGISTRY_DIR="/var/qm/lib/containers/registry"
-export QM_REGISTRY_DIR="/var/lib/containers/registry"
+REGISTRY="quay.io/centos-sig-automotive"
+TOOLS_IMAGE="ffi-tools:latest"
 export QM_TMP_DIR="/var/tmp.dir"
```

Comment:
This one could be deleted
QM_TMP_DIR
QM_IMAGE_TMP_DIR_CONF
CONTAINERS_CONF_DIR